locobot/robot/launch_pyro_habitat.sh

```diff
@@ -2,7 +2,14 @@
 
 export PYRO_SERIALIZER='pickle'
 export PYRO_SERIALIZERS_ACCEPTED='pickle'
+export PYRO_SOCK_REUSE=True
 
+echo "Ensuring clean slate (kills roscore, rosmaster processes)..."
```

Comment:
looks like this is an older commit? this isn't right change. as discussed replace the `pyro-ns` command here.

Reply:
It's odd that that command isn't showing as a diff here. You'll need to rebase your change after the most recent refactor in main - maybe that will fix it. Can you try rebasing and updating this PR? The location of the files in the new structure are [here](https://github.com/facebookresearch/droidlet/blob/main/agents/loco_mc_agent.py#L117) and [here](https://github.com/facebookresearch/droidlet/blob/main/droidlet/lowlevel/locobot/remote/launch_pyro_habitat.sh) and

Reply:
Looks good now!